rasa_core/server.py

```diff
@@ -305,6 +305,16 @@ def update_tracker(sender_id):
         agent().tracker_store.save(tracker)
         return jsonify(tracker.current_state(should_include_events=True))
 
+    @app.route("/domain",
+               methods=['GET'])
+    @cross_origin(origins=cors_origins)
+    @requires_auth(auth_token)
+    @ensure_loaded_agent(agent)
+    def get_domain_yaml():
+        """Get current domain in yaml format."""
+        domain_yaml = agent().domain.to_yaml()
```

Comment:
why are we sending the domain as yaml instead of json?

Reply:
I mean its the way that the domain is stored in the model folder so why not?

Reply:
this is a REST api, so usually the responses are in json format. I mean I don't have a strong opinion on this, but if yaml is used, content negotiation should be used: e.g.

```python
accepts = request.headers.get("Accept", default="application/json")
if accepts.endswith("json"):
  domain = agent().domain.as_dict()
  return jsonify(domain)
elif accepts.endswith("yml"):
  domain = agent().domain.as_yaml()
  return Response(domain_yaml, status=200, content_type="application/x-yml")
else:
  return Response("""Invalid accept header. Domain can be provided as json ("Accept: application/json") or yml ("Accept: application/x-yml"). Make sure you've set the appropriate Accept header.""", status=406)
```